templates/index.html

```diff
@@ -32,7 +32,12 @@ <h3 class="unterschriftweber">
                                     <option value="biblio" data-template="lang:translate">bibliography</option>-->
                                 </select>
                             </div>
-                            <input type="text" class="form-control" placeholder="Standard Suche" name="q" id="query-input"/>
+                            <div data-template="app-shared:if-matches" data-template-key="lang" data-template-value="de">
```

Comment:
Das finde ich ein bisschen umständlich (mit den zwei `<input>` für zwei Sprachen). Kann man den Placeholder-Text nicht auch im `$model` vorbereiten und dann über `app-shared:set-attribute()` reinschreiben?

Reply:
Ja, bitte. Übersetzungen direkt in den HTML templates mache ich nur in Ausnahmen (wenn der Text durch HTML Auszeichnung unterbrochen ist, wie beim Download-Modal), normalerweise bevorzuge ich den Weg über die Dictionary-Dateien. Da ist dann alles an einem Ort

Reply:
Hab es jetzt etwas anders gelöst, aber auch mit nur einem input. :heavy_check_mark: